gradio/external.py

```diff
@@ -42,6 +49,53 @@ def load_blocks_from_repo(name, src=None, api_key=None, alias=None, **kwargs):
     return blocks
 
 
+def get_tabular_examples(model_name) -> Dict[str, List[float]]:
+    readme = requests.get(f"https://huggingface.co/{model_name}/resolve/main/README.md")
```

Comment:
We can either get the example data from the README or the `config.json` but the `config.json` will only have the example data if the model was uploaded with skops. 

I think it would be better if gradio could create a demo for any tabular model and not just those created with skops. Downside is that it introduces a pyyaml dependency.

In the future, once the skops config json file contains richer metadata about feature types (categorical vs null) etc we can read from the config.json if it's present.

Reply:
it was something we didn't want to have in model card specifically 
(cc @adrinjalali is working on having dtypes atm) 
Maybe you could check for both? @freddyaboulton